.github/CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+For guidelines on how to contribute to the KlinicCon Frontend project, please check out our [contribution guide](https://github.com/rt4914/KlinicCon-Backend/wiki). Thank you!
```

Comment:
Please change the text to `KlinicCon Backend project` rather than `KlinicCon Frontend project` and replace the link with the following one:

https://github.com/rt4914/KlinicCon-Backend/wiki#how-to-contribute

Reply:
Fixed. Thanks for your review. @aniketpatidar